community/cypher/cypher-compiler-2.3/src/main/java/org/neo4j/cypher/internal/compiler/v2_3/codegen/CompiledConversionUtils.java

```diff
@@ -28,7 +28,7 @@
 // Class with static methods used by compiled execution plans
 public abstract class CompiledConversionUtils
 {
-    public static boolean isPropertyValueTrue( Object value )
+    public static boolean coerceToPredicate( Object value )
```

Comment:
Maybe `coerceToBoolean`? `coerceToPredicate` sounds like something that should return a function